src/test-utils.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+
+import { render } from './';
+
+export function renderIntoDetachedNode(children: React$Element<any>) {
```

Comment:
This probably doesn't need to be a public thing. react-dom/test-utils should work fine here for this